src/server/game/Combat/ThreatMgr.cpp

```diff
@@ -407,6 +408,20 @@ HostileReference* ThreatContainer::SelectNextVictim(Creature* attacker, HostileR
     return currentRef;
 }
 
+void ThreatContainer::remove(HostileReference *hostileRef)
+{
+    iThreatList.remove(hostileRef);
+
+    sScriptMgr->OnRemoveHostileReferenceFromThreatContainer(hostileRef);
+}
+
+void ThreatContainer::addReference(HostileReference *hostileRef)
```

Comment:
Rename it to AddReference, same goes for the others, always start with captial letter

Reply:
Or maybe you just moved files? Then idc, leave it. Many changes to be made in that case

Reply:
those are not created by me it is core so should i understand it as that when doing anything in core while doing it i should also update surrounding code like this ?

Reply:
You might as well update it while you are in there.